include/ck_tile/ops/grouped_convolution/utils/transform_conv_bwd_weight_to_gemm.hpp

```diff
@@ -422,6 +422,10 @@ struct TransformConvBwdWeightToGemm
 
         // TODO Add support for NumGroupsToMerge > 1
 
+        // return make_naive_tensor_descriptor(make_tuple(N_ * Wo_, K_),
+                                            // make_tuple(NDoHoWoStride, KStride),
```

Comment:
I may be wrong but I suppose that we need to create N * Ho * Wo, K descriptor and after that swap these dimensions. You passed VectorSize and it is probably assigned to last dimension. So last dimension must be continous dimension. But Im not sure so could you check this?